scripts/backfill-stocks.ts

```diff
@@ -0,0 +1,24 @@
+import type { IStock } from "btbot-types";
+
+import { stocks } from "../src/models";
+import { createConnection } from "./connect";
+
+const backFill = [] as IStock[];
+
+export async function main() {
+	await createConnection();
+	const operations = backFill.reduce((acc, stock) => {
```

Comment:
This is kind of strange, how is stick ever null if this is from a backfill

Reply:
Lol no idea, literally just copied what you had for the birthday backfill